docs/receipts-ocr.md

```diff
@@ -0,0 +1,214 @@
+          { "y": 469,  "x": 866  },
+          { "y": 469,  "x": 1753 },
+          { "y": 1413, "x": 1753 },
+          { "y": 1413, "x": 866  }
+        ]
+      },
+      "locale": "pt-PT"
+    },
+    // What folllows is each word found and the location it was found
+    {
+      "description": "Restaurante",
+      "boundingPoly": {
+        "vertices": [
+          // The rectangle where the API found the word
+          { "y": 469, "x": 1009 },
+          { "y": 478, "x": 1187 },
+          { "y": 518, "x": 1185 },
+          { "y": 509, "x": 1007 }
+        ]
+      }
+    },
+    // ... other words here ...
+  ],
+  "2": // Same info for page 2
+}
+```
+
+Most of the time you'll only use the full text provided by
+`2017-02-15-receipts-texts.xz` but, for example, if you want to build some logic
+around analysing the text of specific regions of the receipt you're able to do
+that with the `x` and `y` coordinates above.
+
+## Challenges with OCRing the Chamber of Deputies receipts
+
+There are many challenges in dealing with OCRed info and more specifically for
+the receipts scanned and provided by the Chamber of Deputies we need to keep the
+following in mind:
+
+- Many receipts are filled in by hand and unfortunately handwriting is even
+  harder to parse than "computer text".
+- Some receipts are not scanned carefully and end up being rotated (sometimes it
+  is even scanned up side down).
+- Some scans have really low quality and / or the receipt gets a bit "faded
+  away" due to the thermal paper many places emit, making it harder for the
+  computer to guess what's the text in it.
+
+In other words, there is room for (non trivial) improvements on this OCR process
+(like some image preprocessing) and there are things we can do ourselves in
+order to increase the chances of finding suspicious reimbursements with the PDFs
+provided.
+
+## How can I OCR other receipts?
+
+The the [`ReimbursementOCR`]() class provided by `serenata-toolbox` makes that
+really easy. First you need to install an utility to convert PDFs to images (as
+of this writing the Cloud Vision does not support uploading PDFs directly):
+
+```sh
+# Ubuntu / Debian
+sudo apt-get update && sudo apt-get install poppler-utils
+
+# OSX
+brew install poppler
+```
+
+The following code outlines how one can make use of the functionality provided
+and includes a suggestion on how you can parallelize its execution:
+
+```python
+# Read Google's API key from the `config.ini` file
+import configparser
+settings = configparser.RawConfigParser()
+settings.read('../config.ini')
+google_api_key = settings.get('Google', 'APIKey')
+
+# Configure the client + some dirs for temporary data
+ocr = ReimbursementsOCR('../data/receipts-pdfs', '../data/receipts-imgs')
+
+# Grab the reimbursements you're interested on
+reimbursements = pd.read_csv('../data/2016-12-06-reimbursements.xz', low_memory=False)
+reimbursements = reimbursements.query('some criteria you are interested in')
```

Comment:
Maybe change this "some criteria you are interested in" to just a comment? I'm afraid that people will try to run this whole code block and get an exception.